package.json

```diff
@@ -56,32 +56,32 @@
     "final-form": "^4.11.1",
     "prop-types": "^15.5.10",
     "puppeteer": "^1.11.0",
+    "react": "^16.8.5",
     "react-dom": "^16.8.5",
     "react-final-form": "^6.3.0",
     "react-particles-js": "^2.1.0",
     "react-router-dom": "^5.0.0",
-    "react": "^16.8.5",
```

Comment:
I left a comment in the summary, but when installing from the command line npm automatically alphabetizes the dependencies / devDependencies. This is the reason for these other dependencies moving around.